examples/dt/uni01alpha/README.md

```diff
@@ -0,0 +1,56 @@
+# Deployed Topology uni-alpha
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/commit/7354503e770cbb0435700e2e5b2707de9f7d90e5)**
+
```

Comment:
Above commit link returns 404 Not Found error,
Same applicable to all below files.